proposed/http-client/http-client.md

```diff
@@ -41,12 +41,9 @@ use Psr\Http\Message\ResponseInterface;
 interface ClientInterface
 {
     /**
-     * Sends a PSR-7 request.
-     *
-     * If a request is sent without any prior configuration, an exception MUST NOT be thrown
-     * when a response is recieved, no matter the HTTP status code.
-     *
-     * If a request is sent without any prior configuration, a HTTP client MUST NOT follow redirects.
+     * Sends a PSR-7 request and returns a PSR-7 response. No response will get a special treatment. Ie: 
```

Comment:
> No response will get a special treatment.

Not sure what this means.

Reply:
maybe "Every technically correct HTTP respons MUST be returned as is, even if it represents a HTTP error response or a redirect instruction."

the prosa text should somewhere explain that the PSR client is low level and to guarantee defined behaviour responses may not be acted upon by default (e.g. throwing exceptions for http status codes or redirection). by saying "by default" we do tell that custom configuration, plugins, whatever can violate this.